### PR TITLE
Dark Samus - Charge Shot audio fix and Nair changes

### PR DESCRIPTION
**Pull request by yoonikoo**
Fixed audio to correctly match chargeshot. NAIR has been reverted back to its former on-hit advantage.
[samusdchangelog.txt](https://github.com/HDR-Development/HewDraw-Remix/files/9751535/samusdchangelog.txt)

### CODE REVIEW

**Comment by jobrien97** [comment]
Can you be more specific as to what was fixed with Charge Shot sfx?

**Comment by yoonikoo** [comment]
> Can you be more specific as to what was fixed with Charge Shot sfx?

Currently in nightly, there were reports that the Chargeshot Release SFX did not align with the animation (I believe that Silque had talked to Askew about this). Upon investigation, the SFX was indeed misaligned; so this PR fixes that issue and correctly matches the SFX with animation of the release shot.

File: fighters/samusd/src/acmd/specials.rs [hunk 1]
```diff
@@ -188,12 +188,44 @@ unsafe fn game_specialairlw(fighter: &mut L2CAgentBase) {
     }
 }
 
+#[acmd_script( agent = "samusd_cshot", script = "sound_Shoot", category = ACMD_SOUND, low_priority)]
```
Comment (jobrien97) [inline fighters/samusd/src/acmd/specials.rs]: script name should be in lowercase: `sound_shoot`

**Review by jobrien97 — DISMISSED** [review]
Charge Shot script should be placed in other.rs, after `samusd_cshot_shoot_game`.

File: fighters/samusd/src/acmd/specials.rs (hunk 1 above)
Comment (yoonikoo) [reply fighters/samusd/src/acmd/specials.rs]: Will fix when i get home!

**Review by SkewedAskew — DISMISSED** [review]
Lgtm

**Review by SkewedAskew — APPROVED** [review]
Lgtm... again

**Pull request merged**